kalite/testing/behave_helpers.py

```diff
@@ -70,6 +70,18 @@ def click_and_wait_for_page_load(context, elem, wait_time=MAX_PAGE_LOAD_TIME):
     )
 
 
+def click_and_wait_for_id_to_appear(context, elem_click, elem_wait, wait_time=MAX_WAIT_TIME):
+    """ Click an element and then wait for another element to appear.
+    context: a behave context
+    elem_click: a WebElement to click.
+    elem_wait: an ID selector to wait for.
+    wait_time: Optional. Has a default value.
+    """
+    elem_click.click()
+    id_shown_with_wait(context, elem_wait, wait_time=wait_time)
```

Comment:
What if the element with the given id already exists on the page? Should we try to grab a reference and assert its staleness (if it exists) after the click, in addition to asserting that a *good* reference exists, or shall we say *caveat pragrammor*?